Gemfile

```diff
@@ -24,11 +24,13 @@ gem 'rails_autolink'
 # Please don't update hoptoad_notifier to airbrake.
 # It's for internal use only, and we monkeypatch certain methods
 gem 'hoptoad_notifier', "~> 2.4"
-gem 'draper', :require => false
+gem 'draper'
 
 gem 'errbit_plugin', github: 'errbit/errbit_plugin'
 gem 'errbit_github_plugin', github: 'errbit/errbit_github_plugin'
 
+gem 'dotenv-deployment'
```

Comment:
I'm still thinking this over the dotenv stuff. In the long-term, I don't want to support multiple different configuration paradigms. ENV-based seems way more flexible than the sea-of-YAML approach. So that's a win for dotenv. But we should consider exactly how we want configuration to work. It seems like the only thing we really want out of dotenv-deployment is the one line:
```Dotenv.load('.env')```

It does do some other stuff, but do we want that other stuff?

If we move this direction are we going to provide a .env.example file or something similar? Is that going to be the place where *all* of the configuration goes? Is it going to be required to provide a .env file in development? in production?

We still have other ways of providing configuration (configuration files and passing in environment variables) from outside. What order of precedence should there be if you specify the same config as an ENV, in .env and in a yaml file?